src/components/Article.jsx

```diff
@@ -11,8 +12,17 @@ import Trending from 'components/Trending';
 
 export default class Article extends BaseComponent {
   render () {
+    const meta = [
+      {property: "og:title", content: this.props.title},
+      {property: "og:type", content: "article"},
+      {property: "og:url", content: this.props.url},
+      {property: "og:image", content: this.props.featuredImage},
+      {property: "og:description", content: this.props.teaser},
+      {property: "og:site_name", content: "The Gazelle"},
+    ];
     return (
       <div className="article">
+        <DocMeta tags={meta} />
```

Comment:
Do these tags work if they aren't in the header? I thought they have to be in the header.